src/state/experiments/selectors.js

```diff
@@ -10,10 +11,13 @@ export const getFilteredExperimentsByDate = (state) => {
   const experiments = getExperiments(state); 
   const selectedStartTimestamp = getStartDatepickerTimestamp(state);
   const selectedEndTimestamp = getEndDatepickerTimestamp(state);
+  const status = getStatus(state);
+  const mapStatusToInts = {"rejected": 0, "draft": 1, "review": 2, "ship": 3, "accepted": 4, "live": 5, "complete": 6};
```

Comment:
I just noticed this and updated my last comment to reflect a better way to handle this.

Reply:
Oh, wait does the API return the status as text or an integer?

Reply:
Text! :D

Reply:
Yeah I looked up the API and given that it's text with uppercase first letters. I think you can skip lowercasing these and assume the field is an enum of sorts and use case-matched values:

Building on my earlier comment I think something like this is the correct approach:

**statuses.py**
```js
export const STATUS_REJECTED = "Rejected";
export const STATUS_DRAFT = "Draft";
export const STATUS_REVIEW = "Review";
...

export const STATUSES = {
  STATUS_REJECTED: 0,
  STATUS_DRAFT: 1,
  ...
}
```

**StatusSelector.jsx**
```js
import { STATUSES } from "statuses.js";

<select>
  <option>All</option>
  {Object.keys(STATUSES).map(status => (
    <option value={status} key={status}>
      {status}
    </option>
  ))}
</select>
```

**selectors.js**
```js
const statusPriority = STATUSES[experimentStatus];

return experiments.filter(experiment => {
   ...
   if (status && STATUSES[status] > statusPriority) {
      return false;
   }
});
```

Reply:
Sounds good, I updated the code with a constants.js file! I was also wondering, is it ok to have only: 
export const STATUSES = {
  "Rejected" : 0,
  "Draft" : 1,
  ...
}
in the constants file? I found it easier to find the priority for the experiment status since the API returns it in that format, and it's also in a displayable form to use in the dropdown list. 

I'm not too familiar with best design practices in JS though, so I'm not sure if declaring these consts at the top is a form of best practice: 
export const STATUS_REJECTED = "Rejected";
export const STATUS_DRAFT = "Draft";

:))